index.js

```diff
@@ -139,26 +184,31 @@ const line = []
 
 // ***** Scope & Closures - Question 1 *****
 
-
+let i = 1
+function takeATicketNumber(line) {
+  while (i < line.length) i++
```

Comment:
You have the right idea, but you only need to push `i` into the line once for each function execution!